### PR TITLE
modules/update-service-overview: Simplify intro paragraph

### PR DESCRIPTION
**Pull request by wking**
I'd initially wanted to address "It provides a graph, or diagram that contain" -> "It provides a graph, or diagram, that contains".  But the "of component Operators" bit was confusing to me to.  So I've reworded to lead with "graph of recommended update(s)", and reduce to a single sentence.  I'd also be fine leading with a description of release images, and then introducing the recommended update edges that link them, but that seemed like a bigger refactor.  Wording I'm replacing is from 29016d7cf9 (#12880) and 6e1b894112 (#14991).

/assign @kalexand-rh

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jiajliu, will you PTAL?

**Comment by jiajliu** [comment]
LGTM

File: modules/update-service-overview.adoc [hunk 1]
```diff
@@ -12,11 +12,9 @@
...
 
 The {product-title} update service is the hosted service that provides over-the-air
-updates to both {product-title} and {op-system-first}. It provides a graph,
-or diagram that contain _vertices_ and the _edges_ that connect them, of
-component Operators. The edges in the graph show which versions you can safely
-update to, and the vertices are update payloads that specify the intended state
-of the managed cluster components.
+updates to both {product-title} and {op-system-first}. It serves a graph of
+recommended update _edges_ between {product-title} release image _vertices_ that
+specify the intended state of the managed cluster components.
```
Comment (lbarbeevargas) [inline modules/update-service-overview.adoc:17]: Can you clarify what is specifying "the intended state of the managed cluster components?" It's a little confusing as to what "that" is referring to. For example:

> It serves a graph of recommended update _edges_ between OpenShift Container Platform release image _vertices_. The _edges_ specify the intended state of the managed cluster components.

Also, is "provide" more localization friendly versus "serves?"

Other than these questions/suggestions, LGTM!

**Review by lbarbeevargas — COMMENTED** [review]
A couple of questions/suggestions, but otherwise LGTM!

File: modules/update-service-overview.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/update-service-overview.adoc:17]: The "vertices" specify the intended state of the managed cluster components, and it's important enough that I don't want to diminish it by changing it to "vertices, which..."

@wking, what nuance are you trying to get across with the s/provides/serves? "Serves" is used in the same context in other places in the collection, so I'm inclined to let it stand.

**Pull request closed without merging**